axelor-sale/src/main/java/com/axelor/apps/sale/rest/dto/SaleOrderPostRequest.java

```diff
@@ -30,6 +30,8 @@ public Long getCurrencyId() {
     return currencyId;
   }
 
+  private String inAti;
```

Comment:
We can force the value of inAti to be "true" or "false" with the Pattern annotation.
Look at ManufOrderProductPostRequest for an example.